addons/link_tracker/models/link_tracker.py

```diff
@@ -55,7 +55,9 @@ def convert_links(self, html, vals, blacklist=None):
 
             href = match[0]
             long_url = match[1]
-
+            # Trims UTM values if target is an attachment
+            if '/web/content/' in long_url:
```

Comment:
@chienandalu You can add this in the blacklist list and '/web/content' will not be converted..

Reply:
You'd lose the shortner option that way.